test/parallel/test-path.js

```diff
@@ -507,65 +535,100 @@ assert.strictEqual(path.posix.isAbsolute('./baz'), false);
 
 
 // path.relative tests
-const relativeTests = [
-  [ path.win32.relative,
-    // arguments                     result
-    [['c:/blah\\blah', 'd:/games', 'd:\\games'],
-     ['c:/aaaa/bbbb', 'c:/aaaa', '..'],
-     ['c:/aaaa/bbbb', 'c:/cccc', '..\\..\\cccc'],
-     ['c:/aaaa/bbbb', 'c:/aaaa/bbbb', ''],
-     ['c:/aaaa/bbbb', 'c:/aaaa/cccc', '..\\cccc'],
-     ['c:/aaaa/', 'c:/aaaa/cccc', 'cccc'],
-     ['c:/', 'c:\\aaaa\\bbbb', 'aaaa\\bbbb'],
-     ['c:/aaaa/bbbb', 'd:\\', 'd:\\'],
-     ['c:/AaAa/bbbb', 'c:/aaaa/bbbb', ''],
-     ['c:/aaaaa/', 'c:/aaaa/cccc', '..\\aaaa\\cccc'],
-     ['C:\\foo\\bar\\baz\\quux', 'C:\\', '..\\..\\..\\..'],
-     ['C:\\foo\\test', 'C:\\foo\\test\\bar\\package.json', 'bar\\package.json'],
-     ['C:\\foo\\bar\\baz-quux', 'C:\\foo\\bar\\baz', '..\\baz'],
-     ['C:\\foo\\bar\\baz', 'C:\\foo\\bar\\baz-quux', '..\\baz-quux'],
-     ['\\\\foo\\bar', '\\\\foo\\bar\\baz', 'baz'],
-     ['\\\\foo\\bar\\baz', '\\\\foo\\bar', '..'],
-     ['\\\\foo\\bar\\baz-quux', '\\\\foo\\bar\\baz', '..\\baz'],
-     ['\\\\foo\\bar\\baz', '\\\\foo\\bar\\baz-quux', '..\\baz-quux'],
-     ['C:\\baz-quux', 'C:\\baz', '..\\baz'],
-     ['C:\\baz', 'C:\\baz-quux', '..\\baz-quux'],
-     ['\\\\foo\\baz-quux', '\\\\foo\\baz', '..\\baz'],
-     ['\\\\foo\\baz', '\\\\foo\\baz-quux', '..\\baz-quux'],
-     ['C:\\baz', '\\\\foo\\bar\\baz', '\\\\foo\\bar\\baz'],
-     ['\\\\foo\\bar\\baz', 'C:\\baz', 'C:\\baz']
+const relativeTest = {
```

Comment:
I think it would be best to split out only the tests that would throw.
The other cases need to be tested on all platforms.